.gitignore

```diff
@@ -60,8 +60,6 @@ xcbaselines/
 
 # www
 node_modules/
-# TODO: remove this when assets is moved into build/
-Wikipedia/assets/
```

Comment:
Not sure. It doesn't make sense we would ignore this directory - we need to add files occasionally to it. I don't think it will hurt anything to add it back.